ctadata/api.py

```diff
@@ -58,6 +70,22 @@ def get_endpoint(self, endpoint, path, stream=False, chunk_size=None):
 
         return requests.get(full_url, params=params, stream=stream)
 
+    def webdav4_client(self):
```

Comment:
you are adding another feature except for certificate upload. Ok.
But please rename the PR to indicate these two features.

Reply:
oh, and could you please add something to readme?